via/exceptions.py

```diff
@@ -1,24 +1,10 @@
 """Application specific exceptions."""
 
 from pyramid.httpexceptions import HTTPBadRequest, HTTPConflict, HTTPExpectationFailed
-from requests import exceptions
 
 # pylint: disable=too-many-ancestors
 # It's ok to have a hierarchy of exceptions
 
-REQUESTS_BAD_URL = (
-    exceptions.MissingSchema,
-    exceptions.InvalidSchema,
-    exceptions.InvalidURL,
-    exceptions.URLRequired,
-)
-REQUESTS_UPSTREAM_SERVICE = (
-    exceptions.ConnectionError,
-    exceptions.Timeout,
-    exceptions.TooManyRedirects,
-    exceptions.SSLError,
-)
-
```

Comment:
This keeps these exceptions lists private to where they are used. Not quite sure why these were here if the function wasn't...